pgvector/runtime/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.quarkiverse.langchain4j</groupId>
+    <artifactId>quarkus-langchain4j-pgvector-parent</artifactId>
+    <version>999-SNAPSHOT</version>
+  </parent>
+  <artifactId>quarkus-langchain4j-pgvector</artifactId>
+  <name>Quarkus langchain4j-pgvector - Runtime</name>
+  <description>Do something useful.</description>
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-arc</artifactId>
+    </dependency>
+     <dependency>
+      <groupId>io.quarkiverse.langchain4j</groupId>
+      <artifactId>quarkus-langchain4j-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
@jmartisk @cescoffier I need some help on this, I have added here this dep but for some reasons in my sample project I still need to add this dep.

Reply:
Not sure how you got into this state, but after applying what I suggest in https://github.com/quarkiverse/quarkus-langchain4j/pull/74/files#r1409040930 it works for me, maybe it's related

Reply:
Then the application only needs to depend on `quarkus-langchain4j-pgvector`, nothing else